CHANGELOG.md

```diff
@@ -20,12 +20,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support of core schema API. (#548, #549, #571, #573)
 - Support of `Service#afterCommit(BlockCommittedEvent event)` method
   that is invoked after each block commit event. (#550)
-- Support of Json serialization in a common way. (#611)  
+- Support of Json serialization in a common way. (#611)
+- Added the `--jvm-debug` command line argument that allows JDWP debugging of node. (#629)  
 
 ### Changed
 - `com.exonum.binding.storage.indices.MapEntry` moved to package
   `com.exonum.binding.common.collect`. `FlatMapProof` and `MapIndex` are updated 
   to use this implementation of `MapEntry`.
+- The `--ejb-jvm-args` command line argument has been substituted with `--jvm-args-prepend` and
+ `--jvm-args-append` arguments that are now executed at the `Run` stage instead of 
```

Comment:
I'd also specify that they are not saved in any configuration files unlike `--ejb-jvm-args` used to be.